minima/database.nim

```diff
@@ -20,38 +20,55 @@ type
         KeyNotFound             = "minima: key not found"
         PersistenceFailed       = "minima: persistence failed"
 
-proc log(db: Database, key: seq[byte], value: seq[byte]) =
-    let write = concat(
-        @(uint32(len(key)).toBytes),
-        @(uint32(len(value)).toBytes),
-        key,
-        value
-    )
-
-    # @TODO CATCH EXCEPTIONS
-    discard db.log.writeBytes(write, 0, len(write))
-    db.log.flushFile()
+proc toAESKey*(str: string): array[aes256.sizeKey, byte] =
+    ## Concerts a string to an AES Key for opening an encrypted database.
+    ## 
+    ## **Example::**
+    ## .. code-block::
+    ##   let password = "foo".toAESKey
+    var pass = str
+    var key: array[32, byte]
+    copyMem(addr key[0], addr pass[0], len(pass))
```

Comment:
Presumably here or somewhere else, based on importing `nimcrypto`, it's intended to actually do some encryption?

Reply:
Yeah @tersec, see the EncryptedLog type.